hello-world/src/lib.rs

```diff
@@ -0,0 +1,26 @@
+use tide::{Request, Server};
+
+pub fn create_app() -> Server<()> {
+    let mut app = tide::new();
+    app.at("/hello/:name").get(hello_handler);
+    app
+}
+
+async fn hello_handler(req: Request<()>) -> tide::Result<String> {
+    let name = req.param("name")?;
```

Comment:
Tide seems pretty neat overall but the lack of type-safe routing is a shame 😢

Reply:
`req.param()` is generic over the type though, so I suppose the end result is still type safe (as in you could write `let uid : u32 = req.param("user-id")?;`)

Reply:
Ah yeah I guess it is type-safe in that sense, I just meant though that if you typo `"name"` for example then it will still compile but  it looks like you will just get back a `None` at run-time (or an `Err`..?). Whereas in warp if you have a `String` parameter in the url you are guaranteed to always get a string back

Reply:
hmm, good point.